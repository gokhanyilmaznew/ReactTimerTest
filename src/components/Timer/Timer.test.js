import React from 'react';
import Timer from './Timer';
import renderer from 'react-test-renderer';

describe("Timer component snapshot", () => {
    test("should create snapshot", () => {
    
        expect().toMatchSnapshot();
    });
});