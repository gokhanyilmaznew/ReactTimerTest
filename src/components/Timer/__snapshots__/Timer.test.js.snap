// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer component snapshot should create snapshot 1`] = `undefined`;

exports[`TimerButton component snapshot should create snapshot 1`] = `"test"`;
